data/step_function_inputs/ESACCI_Biomass_L4_AGB_V4_100m_2020.json

```diff
@@ -5,6 +5,7 @@
   "upload": true,
   "user_shared": false,
   "cogify": true,
+  "ingest": false,
```

Comment:
This and the other similar changes stores the input files for the cogify job. I am turning off the ingest here.